.vscode/settings.json

```diff
@@ -3,5 +3,6 @@
     "source.sortImports": true
   },
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
What's the point of this setting?

Reply:
I have no idea? I had to match my typescript version for VSCode to the workspace version and for some reason, VSCode changed that file too? I had to change it because I was constantly getting typescript errors about complex unions and after googling I found out it was because of mismatched typescript versions.

Reply:
Discussed offline that this is prob fine to leave in or we can just delete this file entirely bc we don't need to commit it to begin with

Reply:
I added` /.vscode/` to  `.gitignore ` so that should be good